guides/common/modules/ref_system-requirements.adoc

```diff
@@ -93,7 +93,7 @@ endif::[]
 ifdef::satellite[]
 .Certified hypervisors
 {ProductName} is fully supported on both physical systems and virtual machines that run on hypervisors that are supported to run {RHEL}.
-For more information about certified hypervisors, see https://access.redhat.com/articles/973163[Certified Guest Operating Systems in {OpenStack}, {oVirt}, Red Hat {KubeVirt} and {RHEL} with KVM].
+For more information about certified hypervisors, see https://access.redhat.com/articles/973163[Certified Guest Operating Systems in {OpenStack}, oVirt, Red Hat {KubeVirt} and {RHEL} with KVM].
```

Comment:
```suggestion
For more information about certified hypervisors, see https://access.redhat.com/articles/973163[Certified Guest Operating Systems in {OpenStack}, Red Hat {KubeVirt} and {RHEL} with KVM].
```

Reply:
Maybe provide an updated version just for Satellite with Foreman 3.9+? If you change the article, you'll also change it for earlier Satellite releases. My assumption is that this article is part for 3.7 and below too.

Reply:
That article lists the current supported guest OSes, so it can only be relied upon for the latest version of Satellite. I pinged Martin Tessun. My guess is that he will probably want to remove the RHV sections. In the meantime, we can leave the link alone. I hadn't noticed that I had updated the title of an article. The title is never going to be 'oVirt' because the article is downstream. I will correct the title.